unbxdAutosuggest.js

```diff
@@ -1803,10 +1803,19 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 				+ '{{/if}}';
 		}
 		, prepareHTML: function () {
-			var html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall">',
-				self = this,
-				mainlen = 0,
-				sidelen = 0;
+			var html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall">';
+
+			if (this.options.isMobile) {
+				if (this.options.isMobile()) {
+					html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall mobile">';
```

Comment:
try to use unbxd namespaced classes. replace `mobile` with `unbxd-as-mobile`

Reply:
Done